html/elements/a.json

```diff
@@ -501,19 +501,19 @@
                 "version_added": false
               },
               "opera": {
-                "version_added": false
+                "version_added": "38"
```

Comment:
I show this as landing in 51. https://storage.googleapis.com/chromium-find-releases-static/b5b.html#b5b0ec361f1cc8603d0d772e5cbb7490a0badad8

Reply:
I've got this as landing in Chrome 51. Are you talking about Opera 51? Which would mean Chrome 64, I think?

Reply:
Same for all the HTML attributes?

Reply:
Sorry about that. Chrome 51, Opera 38. I'm not reading carefully enough.